src/openapi.ts

```diff
@@ -18,6 +19,24 @@ const getPathParameters = (koaPath: string) =>
     .filter((part) => part.startsWith(':'))
     .map((part) => part.slice(1));
 
+/**
+ * @param schema json schema object
+ * @return supported openapi schema object type
+ */
+const getSchemaObjectType = (
+  schema: JSONSchema4,
+): OpenAPIV3.NonArraySchemaObjectType => {
+  // TODO supports more types from JSON schema
+  switch (schema.type) {
+    case 'integer':
+      return 'integer';
+    case 'number':
+      return 'number';
+    default:
+      return 'string';
+  }
```

Comment:
I've tried to use explicit typed value but it needs to cover lots of cases so I stopped. This `switch` is for getting the job done as first iteration.